package.json

```
@@ -60,6 +60,7 @@
   "scripts": {
     "build": "gatsby build",
     "develop": "gatsby develop",
+    "dev": "rm -rf ./cache && gatsby develop",
```

```suggestion
    "clean": "rm -rf ./cache",
```

Let's add a `clean` script and call it in another command.